content/blog/container-filesystems/index.md

```diff
@@ -0,0 +1,132 @@
+---
+path: '/container-filesystems'
+title: 'Where are my container''s files? Inspecting container filesystems'
+date: 2021-11-04T06:00:00.000+00:00
+featured_image: hero-image.png
+categories: ['Pixie Team Blogs', 'eBPF']
+authors: ['Omid Azizi']
+emails: ['oazizi@pixielabs.ai']
+featured: true
+---
+
+If you work a lot with containers, then there’s a good chance you’ve wanted to look inside a running container’s filesystem at some point. Maybe the container is failing to run properly and you want to read some logs, maybe you want to check some configuration files inside the container...or maybe you’re like me and want to place some eBPF probes on the binaries in that container (more on this later).
+
+No matter the reason, in this post, we’ll cover a few ways you can use to inspect the files inside a container.
+
+We’ll start with the easy and commonly recommended ways of exploring a container’s filesystem, and talk about why they don’t always work. We’ll follow that up with a basic understanding of how container filesystems are managed by the Linux kernel, and we’ll use that understanding to inspect the filesystem in different, but still easy, ways.
+
+## Method 1: Exec into the container
+
+If you perform a quick search on how to inspect a container’s filesystem, a common solution you’ll find is to use the docker command ([1], [2]). With docker, you can simply run a command like:
```

Comment:
To use the built-in footnote functionality, swap `[1]` --> `[^1]` and then make a footnote section at the bottom with the same.